internal/pkg/describe/env.go

```diff
@@ -50,23 +65,68 @@ func NewEnvDescriber(projectName string, envName string) (*EnvDescriber, error)
 		return nil, fmt.Errorf("assuming role for environment %s: %w", env.ManagerRoleARN, err)
 	}
 	return &EnvDescriber{
-		env:          env,
-		store:        svc,
-		apps:         apps,
-		sessProvider: sess,
+		env:      env,
+		store:    svc,
+		proj:     proj,
+		apps:     apps,
+		rgClient: resourcegroups.New(sess),
 	}, nil
 }
 
 // Describe returns info about a project's environment.
 func (e *EnvDescriber) Describe() (*EnvDescription, error) {
-	var tags map[string]string
+	appsForEnv, err := e.filterAppsForEnv()
+	if err != nil {
+		return nil, err
+	}
+
 	return &EnvDescription{
 		Environment:  e.env,
-		Applications: e.apps,
-		Tags:         tags,
+		Applications: appsForEnv,
+		Tags:         e.proj.Tags,
 	}, nil
 }
 
+func (e *EnvDescriber) filterAppsForEnv() ([]*archer.Application, error) {
+	var appObjects []*archer.Application
```

Comment:
nit:
```suggestion
	var apps []*archer.Application
```

Reply:
Would you mind declaring this variable closer to where it's used? (above the for-loop at l. 109) It just makes it a bit easier for the reader to track how it's used.
https://wiki.c2.com/?DeclareVariablesAtFirstUse